db/testdata/client-cert-with-root-ca.pem

```diff
@@ -0,0 +1,127 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I think they are already removed. Maybe a refresh?

Reply:
I still see some cert files that I think can be deleted in the `db/testdata` directory. Seems like all uses of `ca.pem` were replaced by `ca-ia.pem`, so I think it can be deleted. I didn't find any uses of the following files:

- `mongodb-test-ca.crt`
- `mongodb-test-client.crt`
- `mongodb-test-client-ia.crt`
- `mongodb-test-client-ia.csr`
- `mongodb-test-client-ia.key`
- `mongodb-test-ia.srl`
- `mongodb-test-server1.crt`

Am I missing something here?

Reply:
These files are were used to create the test ca and intermediate certificates. I just wanna keep them in case we will need to create other tests cases, such as another ia from the same ca in the future. They are not directly used for testing. I removed the file that was used to test multi-cert in client cert file.